iceoryx2-ffi/python/README.md

```diff
@@ -0,0 +1,37 @@
+# iceoryx2-ffi-python
+
+## Setup Development Environment
+
+```sh
+# install maturin, see
+# https://github.com/PyO3/maturin
+
+cd $GIT_ROOT
```

Comment:
There should be a line above this with `GIT_ROOT=$(git rev-parse --show-toplevel)`. Does not need to be done is this PR, though.